platform/knative/images/dispatcher/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM ubuntu:20.04
+
+# Copy local code to the container image.
+ENV HOME /dispatcher
+WORKDIR $HOME
+COPY . ./
+
+RUN apt-get update
+RUN apt-get install -y --no-install-recommends apt-utils
+RUN apt-get install -y --no-install-recommends python3 python3-pip
+RUN apt-get install -y --no-install-recommends python3-setuptools python3-wheel
+
```

Comment:
Combine this all into one commmand:
RUN set -ex \
	\
	&& savedAptMark="$(apt-mark showmanual)" \
	&& apt-get update && apt-get install -y --no-install-recommends \
    apt-utils \
    python3 \
    python3-pip \
    python3-setuptools \
    python3-wheel

Reply:
See the Dockerfile in the api folder for an example